flex-config/ui_attributes.common.json

```diff
@@ -132,19 +132,21 @@
         "configuration": [
           {
             "channel": "voice",
-            "auto_accept": false,
-            "auto_select": false,
-            "auto_wrapup": false,
+            "auto_accept": true,
+            "auto_select": true,
+            "auto_wrapup": true,
             "required_attributes": [],
-            "wrapup_time": 30000
+            "wrapup_time": 30000,
+            "default_outcome": "Automatically completed"
           },
           {
             "channel": "chat",
-            "auto_accept": false,
-            "auto_select": false,
-            "auto_wrapup": false,
+            "auto_accept": true,
+            "auto_select": true,
+            "auto_wrapup": true,
             "required_attributes": [],
-            "wrapup_time": 30000
```

Comment:
We should disable this feature by default if we are enabling the automations within it.

Reply:
if you feel strongly lets discuss more - see above.

i feel we want things to be discovered by putting them in the direct path of default behavior.  Undesired behavior can be turned off further driving engagement and understanding of the template admin panel